server/images.go

```diff
@@ -1090,7 +1090,7 @@ func makeRequest(ctx context.Context, method string, requestURL *url.URL, header
 		}
 	}
 
-	req.Header.Set("User-Agent", fmt.Sprintf("ollama/%s (%s %s) Go/%s%s", version.Version, runtime.GOARCH, runtime.GOOS, runtime.Version(), deployment()))
+	req.Header.Set("User-Agent", strings.TrimSpace(fmt.Sprintf("ollama/%s (%s %s) Go/%s %s", version.Version, runtime.GOARCH, runtime.GOOS, runtime.Version(), deployment())))
```

Comment:
This will add a leading space in the event where it's not running in a container. What we might want to do:

```
userAgent := fmt.Sprintf("ollama/%s (%s %s) Go/%s", version.Version, runtime.GOARCH, runtime.GOOS, runtime.Version())

if cp := containerProduct(); cp != "" {
  userAgent += " " + cp
}|
```

or similar